README.md

```diff
@@ -1,45 +1,57 @@
 # SCAAML: Side Channel Attacks Assisted with Machine Learning
+
 ![SCAAML banner](https://storage.googleapis.com/scaaml-public/visuals/scaaml-banner.png)
 
-SCAAML (Side Channel Attacks Assisted with Machine Learning) is a deep learning framework dedicated to side-channel attacks.
-It is written in python and run on top of TensorFlow 2.x.
+SCAAML (Side Channel Attacks Assisted with Machine Learning) is a deep learning
+framework dedicated to side-channel attacks.  It is written in python and run on
+top of TensorFlow 2.x.
 
 [![Coverage Status](https://coveralls.io/repos/github/google/scaaml/badge.svg?branch=main)](https://coveralls.io/github/google/scaaml?branch=main)
 
 ## Available components
 
-- [`scaaml/`](https://github.com/google/scaaml/tree/master/scaaml/): The SCAAML framework code. Its used by the various tools.
-- [`scaaml_intro/`](https://github.com/google/scaaml/tree/master/scaaml_intro): *A Hacker Guide To Deep Learning Based Side Channel Attacks*.
-  Code, dataset and models used in our step by step tutorial on how to use deep-learning to perform AES side-channel attacks in practice.
+-   [`scaaml/`](https://github.com/google/scaaml/tree/master/scaaml/): The
+    SCAAML framework code. Its used by the various tools.
+
+-   [`scaaml_intro/`](https://github.com/google/scaaml/tree/master/scaaml_intro):
+    *A Hacker Guide To Deep Learning Based Side Channel Attacks*.  Code, dataset
+    and models used in our step by step tutorial on how to use deep-learning to
+    perform AES side-channel attacks in practice.
 
 ## Install
 
 ### Dependencies
 
-To use SCAAML you need to have a working version of [TensorFlow 2.x](https://www.tensorflow.org/install) and a version of Python >=3.6
-
+To use SCAAML you need to have a working version of [TensorFlow
+2.x](https://www.tensorflow.org/install) and a version of Python >=3.9
 
 ### SCAAML framework install
 
-1. Clone the repository: `git clone github.com/google/scaaml/`
-2. Create and activate Python virtual environment:
-      `python3 -m venv my_env`
-      `source my_env/bin/activate`
-3. Install dependencies: `python3 -m pip install --require-hashes -r requirements.txt`
-4. Install the SCAAML package: `python setup.py develop`
+1.  Clone the repository: `git clone github.com/google/scaaml/`
+2.  Create and activate Python virtual environment:
+       `python3 -m venv my_env`
+       `source my_env/bin/activate`
+3.  Install dependencies: `python3 -m pip install --require-hashes -r
+    requirements.txt`
+4.  Install the SCAAML package: `python setup.py develop`
 
 ### Update dependencies
```

Comment:
I would add a comment (or move this to a dedicated "Maintainer" section) that this is for maintainers only.
I.e. we don't expect end users to do it